flink-runtime-web/web-dashboard/src/app/interfaces/job-detail.ts

```diff
@@ -50,6 +50,8 @@ export interface JobDetailInterface {
   'start-time': number;
   'end-time': number;
   duration: number;
+  parallelism: number;
+  'job-max-parallelism': number;
```

Comment:
it should be fine to call this `max-parallelism`

Reply:
Done